pkg/config/justification_config.go

```diff
@@ -45,6 +45,9 @@ type JustificationConfig struct {
 	// Issuer will be used to set the issuer field when signing JWTs
 	Issuer string `env:"JVS_API_ISSUER,overwrite,default=jvs.abcxyz.dev"`
 
+	// PluginDir is the directory to load plugins.
+	PluginDir string `env:"JVS_PLUGIN_DIR,overwrite,default=plugins"`
```

Comment:
I would expect this to be an absolute path like `/var/jvs/plugins`

Reply:
[plugin.Discover](https://github.com/hashicorp/go-plugin/blob/a88a423a8813d0b26c8e3219f71b0f30447b5d2e/discover.go#L19) can take both absolute and relative paths, so I was allowing similar things.

Updated this flag to take absolute path instead.

Reply:
I could be wrong but my understanding is that the **default** should be a path like `/vars/jvs/plugins`.

Reply:
You are looking at the stale version actually, that latest version has default value `/vars/jvs/plugins`. Here is the direct link: https://github.com/abcxyz/jvs/pull/294/files#diff-7968fc8c7a029e487f309044142bb04f5dda630cbb12256e474afb8f542cc015

Reply:
I meant we shouldn't require the config to be always absolute. `PluginDir string `env:"JVS_PLUGIN_DIR,overwrite,default=/vars/jvs/plugins"``

Reply:
I don't have enough context to understand major difference between the two choices. @sethvargo and @yolocs could you provide more details on why or why not this should be absolute path?

Reply:
The config tells the main binary to seek plugins in the given dir, which could be anywhere in the system (absolute or not). But _by default_, that _should_ be a absolute path so the location of plugins won't change when you run the binary in different dirs.

Reply:
Thanks Chen for clarification. Updated the code to allow relative overwrite but set the absolute path as default.